tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -111,6 +129,10 @@ function sortRunTableItems(
         continue;
       }
 
+      if (valA === undefined || valB === undefined) {
+        return valB === undefined ? -1 : 1;
+      }
```

Comment:
Just checking: should this also take into account the `sort.direction`?  I don't have a strong opinion, also fine with keeping as is.

Reply:
I intended for undefined to always appear at the end after non-undefined values regardless of the sort direction. It felt very natural to me and I can be convinced otherwise.